lldb/packages/Python/lldbsuite/test/lldbtest.py

```diff
@@ -1360,39 +1328,47 @@ def isARM(self):
             self.getArchitecture().lower().startswith("arm")
         )
 
+    def isSupported(self, cpu_feature: cpu_feature.CPUFeature):
+        triple = self.dbg.GetSelectedPlatform().GetTriple()
+        cmd_runner = self.run_platform_command
+        return cpu_feature.is_supported(triple, cmd_runner)
```

Comment:
I think Python will let you do:
```
        return cpu_feature.is_supported(triple, cmd_runner=self.run_platform_command)
```
Which saves a line.

Reply:
Also pass triple directly instead of making a local variable. It's clear from `GetTriple()` that we're passing a triple to a triple parameter.

Reply:
This formats to:
```
def isSupported(self, cpu_feature: cpu_feature.CPUFeature):
        return cpu_feature.is_supported(
            self.dbg.GetSelectedPlatform().GetTriple(),
            cmd_runner=self.run_platform_command,
        )
```

I prefer the original version for legibility.